driver-core/src/test/resources/unified-test-format/client-side-operation-timeout/sessions-override-operation-timeoutMS.json

```diff
@@ -97,9 +100,6 @@
         {
           "name": "commitTransaction",
           "object": "session",
-          "arguments": {
-            "timeoutMS": 500
```

Comment:
Whats the reasoning for this change?

Reply:
I have reverted this change as we don't have overrides. Thank you for pointing it out!